cypress/global-config/cypress.config.js

```diff
@@ -0,0 +1,33 @@
+const { defineConfig } = require('cypress')
+const { cypressConfig } = require('@axe-core/watcher')
+
+// Get your configuration from environment variables.
+const { API_KEY, SERVER_URL = 'https://axe.deque.com' } = process.env
+
+module.exports = defineConfig(
+  cypressConfig({
+    axe: {
+      apiKey: API_KEY,
+      serverURL: SERVER_URL,
+      /**
+       * configurationOverrides allows users to override the org-wide settings
+       * configured in the axe account.
+       *
+       * Notes:
+       *  1. If you do not wish to override a specific field, simply omit it from this object.
+       *  2. If you lack permission to override a particular field, the configuration will not proceed.
+       *  3. For more details on using global configurations, visit:
+       *     https://docs.deque.com/developer-hub/2/en/dh-global-configuration
+       *  4. For more information on the configurationOverrides object, see:
+       *     https://docs.deque.com/developer-hub/2/en/dh-global-configuration
```

Comment:
I think this is the correct link. If so could you apply these changes to other test examples

```suggestion
       *     https://docs.deque.com/developer-hub/2/en/dh-api-reference#configurationoverrides-interface
```